algebird-test/src/test/scala/com/twitter/algebird/TupleAggregatorsTest.scala

```diff
@@ -334,4 +334,638 @@ class TupleAggregatorsTest extends WordSpec with Matchers {
       assert(agg(longData) == Tuple22(6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6))
     }
   }
+
+  "MapAggregator" should {
+    import MapAggregator._
+
+    "Create an aggregator from 2 (key, aggregator) pairs" in {
```

Comment:
these tests would be more informative if there were different aggregators used for each key. It would also be a better test if the aggregators had different intermediate types (like, the `B` in the agg's A, B, C type params).